### ISSUE TITLE
can't get loss below 5.545

### ISSUE THREAD

**Issue by dlahoti** [ibab/tensorflow-wavenet#136]
For some reason, I can't get a loss below 5.545¸ regardless of options or dataset. Generating with the trained models just gives me static.

**Comment by jyegerlehner** [comment]
If you're using defaults, you need to decrease the learning rate to 0.001 just to get it learning something. I think one also needs larger receptive field than what's in the default wavenet_params.json. 

There're example params files over in the issue thread on "Generating good audio samples".

Note to self: PR to change those defaults.

**Comment by ibab** [comment]
Yeah, we should definitely change the broken defaults :)

**Comment by ibab** [comment]
We have better defaults now, and @jyegerlehner also fixed a different issue that was causing instability in the optimization, so I would consider this fixed now.
